page_objects/core_helpers/selenium_action_objects.py

```diff
@@ -290,9 +290,18 @@ def drag_and_drop(self, source_locator, target_locator):
 
         return result_flag
 
-    def teardown(self):
+    def teardown(self, test_status = None):
         "Tears down the driver"
-        self.driver.quit()
-        self.reset()
-
-
+        import os
+        if os.getenv('REMOTE_BROWSER_PLATFORM') == 'LT' and test_status is not None:
```

Comment:
Do we need to modify the existing `teardown` method to use `execute_script` method?
Can we:
1. Add `self.driver` to `LambdaTestRunner` object and implement a method to update `lambda-status`?
2. Use `testobj.driver` in `conftest.py` to run `execute_script` method?

Reply:
Done.